content/millennium/r4/documents/binary.md

```diff
@@ -0,0 +1,42 @@
+---
+title: Binary | R4 API
+---
+
+# Binary
+
+* TOC
+{:toc}
+
+## Overview
+
+The Binary resource can contain any clinical content such as text, images, and PDFs.  This resource is currently limited to clinical documents and diagnostic reports.  The diagnostic reports are currently limited to Anatomic Pathology and Radiology.
+
+It is recommended to request all Binary resources only after obtaining a link to the resource via references from DiagnosticReport or DocumentReference. It is not recommended to start a workflow in the Binary resource. 
+
+The consumer must populate the Accept header with either application/fhir+json or the format returned in the attachment.contentType of the referring resource.  If the Accept header is application/fhir+json, a FHIR Binary resource is returned with the raw data populated in the content. Otherwise, the raw data will be returned (not contained within a FHIR resource). For more information, see [the Binary Documentation](http://hl7.org/fhir/r4/binary.html#rest).
+
+NOTE: To retrieve documents with application/pdf contentType, the client must have installed Cerner’s Clinical Reporting (XR) solution. For additional information about Cerner’s Clinical Reporting (XR) solution see [Clinical Reporting (XR) Reference Pages](https://wiki.ucern.com/display/reference/Clinical+Reporting+XR+Reference+Pages).
+
+The following fields are returned if valued:
+
+* [Binary id](http://hl7.org/fhir/r4/resource-definitions.html#Resource.id){:target="_blank"}
+* [Content type](http://hl7.org/fhir/r4/binary-definitions.html#Binary.contentType){:target="_blank"}
+* [Content](http://hl7.org/fhir/r4/binary-definitions.html#Binary.content){:target="_blank"}
+
+## Retrieve by id
+
+List an individual Binary by its id:
+
+    GET /Binary/:id
+
+_Implementation Notes_
+
+* This is usually linked from DocumentReference or DiagnosticReport and should generally be accessed using the exact link given in the referring resource. Modifying the link has undefined consequences.
+* Documents containing URL sections are unsupported with contentType of `application/pdf`
+* See the [headers](#headers) section for concerns about the Accept header.
+* See the [authorization](#authorization-types) section for concerns about the required OAuth scopes.
+
+### Authorization Types
+
+  The Binary.read scope and either the DocumentReference.read or DiagnosticReport.read scopes are required.
+  Practitioner | System
```

Comment:
* Practitioner and System are supposed to be linked to other sections of the documentation
* The following sections need to be added: Headers, Example, and Errors

See [Practitioner](http://fhir.cerner.com/millennium/r4/individuals/practitioner/#retrieve-by-id) for an example.

Reply:
Updated @averyallen  https://github.com/cerner/fhir.cerner.com/commit/4e4c61099afb22bb16dca0268cef934957bc4bc6

Reply:
this needs work yet. the authorization types section must be updated

Reply:
https://github.com/cerner/fhir.cerner.com/commit/70b171a1701f4afd9b85d8b10f6a8a609678eb5f#diff-aaa0de638fb809d6747b985beb976128R41

Reply:
please do this:
```
<%= authorization_types(practitioner: true, patient: false, system: true) %>

_Implementation Note_

The `Binary.read` scope and either the `DiagnosticReport.read` or the `DocumentReference.read` scope is required.
```

Please further specify how a consumer knows whether they should set the DiagReport scope or the DocRef scope

Reply:
@MaxPhilips in dstu2 it is 
```
The `Binary.read` scope and either the `DiagnosticReport.read` or the `DocumentReference.read` scope is required.

<%= authorization_types(practitioner: true, patient: false, system: true) %>
```

Do you want me to flip it.

Reply:
https://github.com/cerner/fhir.cerner.com/commit/b059a8899514f91baa449669f35f9889fb89d5bf

Reply:
yes I want you to flip it

Reply:
can you also include the backticks like in the examples above?

Reply:
Added ticks and @max it was already flipped.